core/src/main/resources/lib/layout/notice.jelly

```diff
@@ -31,10 +31,18 @@ THE SOFTWARE.
     <st:attribute name="icon" use="required">
       Sets the icon for the notice. Supports Jenkins Symbols and regular images.
     </st:attribute>
+    <st:attribute name="description">
+      Optional description for the notice. HTML is supported.
+    </st:attribute>
```

Comment:
Why is this an attribute rather than a body?

Reply:
Body makes more sense 👍 Think I was tunnel visioned by the other attributes and hadn't considered the sensible choice. Updated it, thanks!